Cargo.toml

```diff
@@ -2,6 +2,7 @@
 authors = ["Corentin Henry <corentinhenry@gmail.com>"]
 name = "xi-term"
 version = "0.1.0"
+edition = "2018"
```

Comment:
:+1: I should have made this the default already... Thanks.

Reply:
I'm already so used to the 2018 edition that I actually stumbled a bit, trying to remember "how this crates and modules stuff worked again in the old edition" :-)